docs/pages/docs/ci/gitlabci.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Using Turborepo with GitLab CI
+description: How to use GitLab CI with Turborepo to optimize your CI workflow
+---
+
+# Using Turborepo with GitLab CI
+
+The following example shows how to use Turborepo with [GitLab CI](https://docs.gitlab.com/ee/ci/) and [pnpm](https://pnpm.io).
+
+Create a `.gitlab-ci.yml` file:
+
+```yaml
+image: node:latest
+stages:
+  - build
+build:
+  stage: build
+  before_script:
+    - curl -f https://get.pnpm.io/v6.16.js | node - add --global pnpm@7.0.0-rc.9
```

Comment:
Specific reason to use v7?

Reply:
Copied and pasted directly from pnpm

Reply:
Curious why they promote a release candidate. Think the `--filter` argument has some changes which might not yet be supported by Turbo yet so could cause some confusion